packages/nuxt/src/app/components/nuxt-stubs.ts

```diff
@@ -0,0 +1,17 @@
+import { createError } from '../composables/error'
+
+function renderStubMessage (name: string) {
+  throw createError({
+    fatal: true,
+    statusCode: 500,
+    statusMessage: `${name} is provided by @nuxt/image. Check your console to install it or run 'npx nuxi@latest module add @nuxt/image'`
```

Comment:
Is this `... add image` as in the [PR](https://github.com/nuxt/nuxt/pull/23741/files#diff-56eb09419c0f270f530ddd38f7161f0d157341991971611ac7fb060db83d97efR19) or `... add @nuxt/image`?

Reply:
Either way works.